src/applications/vaos/project-cheetah/index.jsx

```diff
@@ -56,6 +73,10 @@ export function NewBookingSection({ featureProjectCheetah }) {
 
   return (
     <FormLayout>
+      {(windowsStatus === FETCH_STATUS.loading ||
+        windowsStatus === FETCH_STATUS.notStarted) && (
+        <LoadingIndicator message="Checking COVID-19 vaccine availability" />
```

Comment:
We probably should be careful with this message. We're checking to see if you can make an online appointment for the vaccine, not if there's any available.

Reply:
Modified to 'Checking for online appointment availability'.